packages/suite-data/files/connect/workers/shared-connection-worker.4e3e8b51bdda80805b28.js

```diff
@@ -0,0 +1 @@
d({id:t,message:r},{postMessage:n})};var r=t(1);"undefined"!=typeof onconnect&&(onconnect=function(e){var n=e.ports[0];n.onmessage=function(e){d(e.data,n)}});var o={},i={},u=null,s=Promise.resolve();function c(){var e=(0,r.create)();u=e,setTimeout((function(){return e.reject(new Error("Timed out"))}),1e4)}function f(e){null!=u&&u.resolve(e)}function a(){return null==u?Promise.reject(new Error("???")):u.promise}function l(e){var n=s.then((function(){return e()}));s=n.catch((function(){}))}function d(e,n){var t=e.id,r=e.message;if("acquire-intent"===r.type){var u=r.path,s=r.previous,d=r.debug;l((function(){return function(e,n,t,r,u){var s=!1,f=t?i:o,l=t?o:i,d=f[e];s=null==d?null!=n:n!==d;return s?(m({type:"wrong-previous-session"},r,u),Promise.resolve()):(c(),m({type:"other-session",otherSession:l[e]},r,u),a().then((function(n){if(n.good){var r=(++y).toString();t&&(r="debug"+r),f[e]=r,m({type:"session-number",number:r},n.id,u)}else m({type:"ok"},n.id,u)})))}(u,s,d,t,n)}))}if("acquire-done"===r.type&&function(e){f({good:!0,id:e})}(t),"acquire-failed"===r.type&&function(e){f({good:!1,id:e})}(t),"get-sessions"===r.type&&l((function(){return v(t,n)})),"get-sessions-and-disconnect"===r.type){var g=r.devices;l((function(){return v(t,n,g)}))}if("release-onclose"===r.type){var b=r.session;l((function(){return function(e){var n=null;if(Object.keys(o).forEach((function(t){o[t]===e&&(n=t)})),null==n)return Promise.resolve();var t=n;return delete o[t],delete i[t],Promise.resolve()}(b)}))}if("release-intent"===r.type){var h=r.session,P=r.debug;l((function(){return function(e,n,t,r){var u=null,s=n?i:o,f=n?o:i;if(Object.keys(s).forEach((function(n){s[n]===e&&(u=n)})),null==u)return m({type:"double-release"},t,r),Promise.resolve();var l=u,d=f[l];return c(),m({type:"path",path:l,otherSession:d},t,r),a().then((function(e){delete s[l],m({type:"ok"},e.id,r)}))}(h,P,t,n)}))}"release-done"===r.type&&p(t),"enumerate-intent"===r.type&&l((function(){return function(e,n){return c(),m({type:"ok"},e,n),a().then((function(e){m({type:"ok"},e.id,n)}))}(t,n)})),"enumerate-done"===r.type&&p(t)}function p(e){f({id:e})}function v(e,n,t){if(null!=t){var r={};t.forEach((function(e){r[e.path]=!0})),Object.keys(o).forEach((function(e){o[e]||delete o[e]})),Object.keys(i).forEach((function(e){i[e]||delete i[e]}))}return m({type:"sessions",debugSessions:i,normalSessions:o},e,n),Promise.resolve()}var y=0;function m(e,n,t){t.postMessage({id:n,message:e})}},function(e,n,t){"use strict";Object.defineProperty(n,"__esModule",{value:!0}),n.create=function(){var e=function(e){},n=function(e){},t=new Promise((function(t,r){e=t,n=r})),r=t.then((function(){throw new Error("Promise is always rejecting")}));return r.catch((function(){})),{resolve:e,reject:n,promise:t,rejectingPromise:r}},n.resolveTimeoutPromise=function(e,n){return new Promise((function(t){setTimeout((function(){t(n)}),e)}))},n.rejectTimeoutPromise=function(e,n){return new Promise((function(t,r){setTimeout((function(){r(n)}),e)}))}}]);
```

Comment:
I'm wondering if it's a good idea to track this (and iframe.js) in git. They're generated from connect in `node_modules` which is pinned by its version. This means that when someone is bumping connect version in suite they also must remember to re-build connect here in `suite-data` and commit the results. That's easily forgotten and can lead to bugs.

Can't we just generate these files on `yarn build:lib` and stop tracking them in git?

Reply:
they are on time generated using `yarn build:connect`
i dont want to rewrite the whole stack right now

Reply:
Not sure that you mean by `on time generated`.

Anyway, I'll resolve this as out of scope of this PR. We can do that later.

Reply:
https://github.com/trezor/trezor-suite/blob/f19f9e36e1526df4b11eaebecb983891b963423d/docs/packages/connect/bump.md

Reply:
@goodhoko could you open an issue pls? I agree it would be better not to have it in git.

Reply:
@tsusanka here you go https://github.com/trezor/trezor-suite/issues/3734